### PR TITLE
Contxtful RTD Provider : add ORTB2 support

### PR DESCRIPTION
**Pull request by sebastienrufiange**
## Type of change
- [ x ] Feature

## Description of change
- Injects the receptivity metric in the ORTB2 object for the configured bidders
- Leverages session storage for low-latency receptivity injections
- Updated documentation

## Other information
Documentation [PR](https://github.com/prebid/prebid.github.io/pull/5322)

### CODE REVIEW

File: modules/contxtfulRtdProvider.md [hunk 1]
```diff
@@ -35,31 +52,82 @@ pbjs.setConfig({
...
-|------------|----------|----------|-------------------------------------------|
-| `version`  | `string` | Required | Specifies the API version of Contxtful.   |
-| `customer` | `string` | Required | Your unique customer identifier.          |
+| Name                | Type     | Scope    | Description                                |
+|---------------------|----------|----------|--------------------------------------------|
+| `version`           | `String` | Required | Specifies the version of the Contxtful     |
+|                     |          |          | Receptivity API.                           |
+| `customer`          | `String` | Required | Your unique customer identifier.           |
+| `hostname`          | `String` | Optional | Target URL for CONTXTFUL external JavaScript file. Default is "api.receptivity.io". Changing default behaviour is not recommended. Please reach out to contact@contxtful.com if you experience issues. |
+| `adServerTargeting` | `Boolean`| Optional | Enables the `getTargetingData` to inject targeting value in ad units. Setting this parameter to true enables the feature. [Default] Setting this parameter to false disables the feature.       |
```
Comment (AlexDesBen) [inline modules/contxtfulRtdProvider.md:74]: Same comment as PR https://github.com/prebid/prebid.github.io/pull/5322

File: modules/contxtfulRtdProvider.md [hunk 2]
```diff
@@ -35,31 +52,82 @@ pbjs.setConfig({
...
-This module also integrates seamlessly with Google Ad Manager, ensuring that the `ReceptivityState` is available as early as possible in the ad serving process.
\ No newline at end of file
+This module also integrates seamlessly with Google Ad Manager, ensuring that the `ReceptivityState` is available as early as possible in the ad serving process.
+
+## Usage: Injection in ortb2 for bidders
+
+Setting the `bidders` field in the configuration parameters enables Receptivity in the `ortb2` object through `getBidRequestData` for all the listed bidders.
+On a Bid Request Event, all bidders in the configuration will inherit the Receptivity data through `ortb2`
+Default is `[]` (an empty array)
+RECOMMENDED : Add all the bidders active like this `["bidderCode1", "bidderCode", "..."]`
```
Comment (AlexDesBen) [inline modules/contxtfulRtdProvider.md:95]: Same comment as PR https://github.com/prebid/prebid.github.io/pull/5322

Comment (sebhtml) [reply modules/contxtfulRtdProvider.md:96]: Done. see https://github.com/prebid/Prebid.js/pull/11497/commits/73a2c409e547357371235a1d3094915a4c26632f.

File: modules/contxtfulRtdProvider.md (hunk 1 above)
Comment (sebhtml) [reply modules/contxtfulRtdProvider.md]: Done. see https://github.com/prebid/Prebid.js/commit/73a2c409e547357371235a1d3094915a4c26632f.

**Review by patmmccann — APPROVED** [review]

**Pull request merged**